source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -5,12 +5,293 @@
 Data management
 ===============
 
-The Data Management component includes classes and utilities for data acquisition, initial preprocessing
-and normalization, for data conversion into numeric formats (performed by one of supported Data Sources)
-and for model representation.
+This section includes descriptions of concepts and objects that operate with
+data. For |dal_short_name|, such set of operations, or **data management**, is
+distributed among different stages of a :ref:`data analytics pipeline
+<data_analytics_pipeline>`. From data management angle of view, this pipeline
+contains three main steps of data acquisition, preparation, and computation:
+
+1. Raw data acquisition.
+
+  - Transferring out-of-memory data into in-memory representation from sources
+    like databases, files, remote storages, etc.
+
+2. Data preparation.
+
+  - Performing data normalization, filtering.
+  - Recovery of missing values
+  - Compression and decompression of the data.
+  - Conversion of data to numeric representation.
+  - Supporting different in-memory :term:`data formats <Data format>`.
+  - Computation of statistics on numerical data like mean, variance,
+    covariance, etc.
+
+3. Algorithm computation.
+
+  - Streaming in-memory numerical data to the algorithm.
+
+In complex usage scenarios, data flow can go through these three stages back and
+forth. For example, when the full data are not available at the start of the
+computation, it can be done step-by-step with blocks of data. After computation
+on the current block the next one should be obtained and prepared.
+
+.. image:: _static/data_management_flow.png
+  :width: 800
+  :alt: Typical data management flow
+
+Key concepts
+============
+
+|dal_short_name| provides a set of concepts to operate on out-of-memory and
+in-memory data during different stages of the :ref:`data analytics pipeline
+<data_analytics_pipeline>`.
+
+.. _dataset:
+
+Dataset
+--------
+
+The most common concept over the data in |dal_short_name| is a :term:`dataset`.
```

Comment:
`most common` -> most widely used? the main concept?
`concept over the data` - incorrect phrase

suggestion:

```
The main data-related concept that |dal_short_name| works with is a :term:`dataset`.
```

Reply:
Thanks!